src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -651,7 +651,8 @@ public void testEnrollStudents() throws Exception {
         verifyEnrollmentResultForStudent(saf.makeStudent(line4, courseIdForEnrollTest),
                                          enrollResults.studentList.get(4), StudentUpdateStatus.NEW);
             
-        CourseDetailsBundle cd = coursesLogic.getCourseDetails(courseIdForEnrollTest);
+        CourseDetailsBundle cd = coursesLogic
+                .getCourseSummary(courseIdForEnrollTest);
```

Comment:
I tried to follow JCC, where max line size - 80, but now I see that you use code convention from google, where limit per line - 100 symbols.